### PR TITLE
[Samples][Java] Fix disparities of 45.state-management sample

### PR DESCRIPTION
**Pull request by Batta32**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

**Review by FedericoBernal — APPROVED** [review]

File: samples/java_springboot/45.state-management/README.md [hunk 1]
```diff
@@ -1,88 +1,135 @@
...
 
 After the bot is deployed, you only need to execute #6 if you make changes to the bot.
 
+## Bot State
 
-## Further reading
+A key to good bot design is to track the context of a conversation, so that your bot remembers things like the answers to previous questions. Depending on what your bot is used for, you may even need to keep track of state or store information for longer than the lifetime of the conversation. A bots state is information it remembers in order to respond appropriately to incoming messages. The Bot Builder SDK provides classes for storing and retrieving state data as an object associated with a user or a conversation.
 
+## Further reading
 - [Maven Plugin for Azure App Service](https://docs.microsoft.com/en-us/java/api/overview/azure/maven/azure-webapp-maven-plugin/readme?view=azure-java-stable)
```
Comment (fran893) [inline samples/java_springboot/45.state-management/README.md]: This link points to a 404 - Content not found

File: samples/java_springboot/45.state-management/src/main/java/com/microsoft/bot/sample/statemanagement/Application.java [hunk 2]
```diff
@@ -3,9 +3,10 @@
 
 package com.microsoft.bot.sample.statemanagement;
 
-import com.microsoft.bot.builder.Bot;
-import com.microsoft.bot.builder.ConversationState;
-import com.microsoft.bot.builder.UserState;
+import com.microsoft.bot.azure.CosmosDbPartitionedStorage;
+import com.microsoft.bot.azure.CosmosDbPartitionedStorageOptions;
+import com.microsoft.bot.azure.blobs.BlobsStorage;
+import com.microsoft.bot.builder.*;
```
Comment (fran893) [inline samples/java_springboot/45.state-management/src/main/java/com/microsoft/bot/sample/statemanagement/Application.java]: We need to replace this import with individuals imports, right?

**Review by fran893 — CHANGES_REQUESTED** [review]

File: samples/java_springboot/45.state-management/README.md (hunk 1 above)
Comment (Batta32) [reply samples/java_springboot/45.state-management/README.md]: Thanks for that

File: samples/java_springboot/45.state-management/src/main/java/com/microsoft/bot/sample/statemanagement/Application.java (hunk 2 above)
Comment (Batta32) [reply samples/java_springboot/45.state-management/src/main/java/com/microsoft/bot/sample/statemanagement/Application.java]: Thanks for that

**Review by fran893 — APPROVED** [review]

**Pull request closed without merging**